application/models/file.go

```diff
@@ -287,3 +287,9 @@ func (f *File) ClearLinked() error {
 	f.Linked = false
 	return DB.UpdateColumns(f, "linked")
 }
+
+// FindByIDs finds all Files associated with the given IDs and loads them from the database
+func (f *Files) FindByIDs(ids []int) error {
+	ids = domain.UniquifyIntSlice(ids)
```

Comment:
If this `UniquifyIntSlice` isn't done, PostgresQL will de-dupe them. Are you thinking this is better for performance?

Reply:
Are you saying Postgres will de-dupe the results or the integers going into the query?  If it's the results that get de-duplicated, wouldn't this function speed it up by reducing the numbers of integers in the where clause?

I don't really know myself.  It just looks like pop doesn't do any de-duping in advance.

Reply:
I don't know about Pop or lib/pg but Postgres does de-dupe the results. I think you're right in that it would help with performance because the size of the query would be smaller, but maybe negligible. Just wanted to point it out in case you wanted to simplify. No problem either way.